zkvm/precompiles/src/secp256k1.rs

```diff
@@ -61,8 +84,15 @@ pub fn verify_signature(
             let pubkey_x = Scalar::from_repr(bits2field::<Secp256k1>(&pubkey[1..33]).unwrap()).unwrap();
             let pubkey_y = Scalar::from_repr(bits2field::<Secp256k1>(&pubkey[33..]).unwrap()).unwrap();
 
+            let mut pubkey_x_le_bytes = pubkey_x.to_bytes();
+            pubkey_x_le_bytes.reverse();
+            let mut pubkey_y_le_bytes = pubkey_y.to_bytes();
+            pubkey_y_le_bytes.reverse();
```

Comment:
why did you need to add this?

Reply:
The earlier implementation of `AffinePoint` was based on the `Scalar` type, meant for the `Secp256k1` curve. To accommodate both `Bn254` and `Secp256k1` curves in a more generic manner, the representation of `AffinePoint` had to be adapted to use `u8` limbs in little-endian order.

Reply:
I'm trying to covert the Scalar into little endian byte format here